corehq/apps/smsforms/models.py

```diff
@@ -35,6 +35,22 @@ class SQLXFormsSession(models.Model):
     # Maximum session length of 7 days
     MAX_SESSION_LENGTH = 7 * 24 * 60
 
+    STATUS_PARTIALLY_COMPLETE = "partially_completed"
+    STATUS_COMPLETE = "completed"
+    # "In Progress" means the survey session is open and the survey can still
+    # be completed by the user
+    STATUS_IN_PROGRESS = "in-progress"
+    # "Not Finished" means the survey session closed without generating a form submission
+    # and the survey can no longer be completed
+    STATUS_NOT_FINISHED = "not-finished"
```

Comment:
@calellowitz FYI Giovanni responded with the distinction between 'in-progress' and 'not-finished'. I've added comments here to explain.